brave/brave-core
# Adds different logo for verified publishers

Resolves https://github.com/brave/brave-browser/issues/4043

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on

## Test Plan:
- enable rewards
- don't dismiss grant notification
- go to non-verified site
- make sure that bat logo is the same
- go to verified site
- make sure that you see regular logo with notification count
- dismiss notification
- make sure that you see verified bat logo
- go to non-verified site and make sure that you see regular logo



## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
